pkgs/development/compilers/gambit/bootstrap.nix

```diff
@@ -1,6 +1,6 @@
-{ stdenv, fetchurl, autoconf, gcc, coreutils, ... }:
+{ gccStdenv, lib, fetchurl, autoconf, gcc, coreutils, ... }:
```

Comment:
`gccStdenv` is for Darwin reasons, right?

Reply:
I'm adding this comment:
```
# As explained in build.nix, GCC compiles Gambit 10x faster than Clang, for code 3x better
```